gatsby-ssr.js

```diff
@@ -26,10 +26,14 @@ var getFonts = function getFonts(options) {
     .replace(/ /g, '+')
 }
 
+var getDisplay = function getDisplay(options) {
```

Comment:
I think this would be sufficient
```suggestion
function getDisplay(options) {
```

Reply:
Done.